src/Storage/Storage.Management/help/New-AzStorageQueueSASToken.md

```diff
@@ -135,7 +135,7 @@ Accept wildcard characters: False
 
 ### -Permission
 Specifies permissions for a storage queue.
-It is important to note that this is a string, like `rwd` (for Read, Write and Delete).
```

Comment:
Please update it in [NewAzureStorageQueueSasToken.cs](https://github.com/Azure/azure-powershell/blob/main/src/Storage/Storage/Queue/Cmdlet/NewAzureStorageQueueSasToken.cs) line 61. And regenerate the help md files with this [guide](https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/help-generation.md#updating-help-after-making-cmdlet-changes).

Reply:
@YanaXu Have tried the commands in the guide, but they didn't work for the previously customized help message. Thus we've been updating the customized description manually

Reply:
Hi @yifanz7, I'll merge this PR now since it's the code complete date for this sprint. Please start a new PR to update the md files in a correct workflow:

- Update the help message in [NewAzureStorageQueueSasToken.cs](https://github.com/Azure/azure-powershell/blob/main/src/Storage/Storage/Queue/Cmdlet/NewAzureStorageQueueSasToken.cs) line 61
- Remove the Parameter definition in src/Storage/Storage.Management/help/New-AzStorageQueueSASToken.md.  (From line 136 to line 201 in this md file.)
- Run **Update-MarkdownHelpModule** to re-generate the md file.